plugins/wazuh-core/server/services/initialization/types.ts

```diff
@@ -0,0 +1,39 @@
+import { LifecycleService } from '../types';
+
+export interface InitializationTaskDefinition {
+  name: string;
+  run: (ctx: any) => any;
+}
+
+export interface InitializationTaskRunData {
+  name: InitializationTaskDefinition['name'];
+  status: 'not_started' | 'running' | 'finished';
+  result: 'success' | 'fail' | null;
+  createdAt: string | null;
+  startedAt: string | null;
+  finishedAt: string | null;
+  duration: number | null; // seconds
+  data: any;
+  error: string | null;
+}
+
+export interface IInitializationTask extends InitializationTaskRunData {
```

Comment:
```suggestion
export interface InitTaskRunner extends InitializationTaskRunData {
```

Reply:
I will keep the original name of the interface.